src/main/resources/templates/modules_import.html

```diff
@@ -76,19 +76,19 @@
                 </template>
             </b-card>
         </div>
-<!--        <div class="col-md-4">-->
-<!--            <b-card no-body>-->
-<!--                <b-button size="lg" variant="outline-secondary" class="btn-gitlab" squared @click="toggle('gitlab')">-->
-<!--                    <i class="fab fa-gitlab"></i> from Gitlab-->
-<!--                </b-button>-->
-
-<!--                <repository-import v-if="selected == 'gitlab'" :registry="'gitlab'"></repository-import>-->
-
-<!--                <template v-slot:footer>-->
-<!--                    <em>Import an existing module code from existing Gitlab repository</em>-->
-<!--                </template>-->
-<!--            </b-card>-->
-<!--        </div>-->
+        <div class="col-md-4" th:if="${user.oAuth2User.provider == 'gitlab'}">
```

Comment:
Is it possible to extract a common vue component for github/gitlab import ?

Reply:
done ✔️